torchbiggraph/converters/import_from_tsv.py

```diff
@@ -253,9 +225,8 @@ def convert_input_data(
     entity_path: str,
     edge_paths_out: List[str],
     edge_paths_in: List[Path],
-    lhs_col: int,
-    rhs_col: int,
-    rel_col: Optional[int] = None,
+    edgelist_reader: EdgelistReader,
+    edgelist_format: str = "tsv",
```

Comment:
What's this argument's purpose? It's not used, and not passed by the caller...

Reply:
Oops, left this in by accident.